Packs/MalwareBazaar/Integrations/MalwareBazaarFeed/MalwareBazaarFeed_description.md

```diff
@@ -0,0 +1 @@
+Fetch indicators from a MalwareBazzaar feed. You can apply filters to the feed to focus which indicators you want to ingest.
```

Comment:
usually used to have info regarding the authentication or different fetch filters. please add this info instead with examples, as this is mainly used for setup purposes, and not general info.

Reply:
e.g:
https://github.com/demisto/content/blob/master/Packs/Zimperium/Integrations/Zimperium/Zimperium_description.md
https://github.com/demisto/content/blob/master/Packs/QRadar/Integrations/QRadar_v2/QRadar_v2_description.md#query-to-fetch-offenses